src/const.ts

```diff
@@ -32,6 +32,293 @@ export const CONTAINER_ROOT_DIR = '/data';
 // https://docs.docker.com/desktop/features/networking/#use-cases-and-workarounds
 export const CONTAINER_LOCAL_HOSTNAME = 'host.docker.internal';
 
+export const DEFAULT_CONFIG_FILENAME = 'vivliostyle.config.js';
+export const DEFAULT_PROJECT_TITLE = 'My Book Title';
+export const DEFAULT_PROJECT_AUTHOR = 'John Doe';
```

Comment:
```suggestion
export const DEFAULT_PROJECT_TITLE = 'My Title';
export const DEFAULT_PROJECT_AUTHOR = 'My Name';
```

Vivliostyleの使用目的はBookに限らず、レポートのようなものかもしれないので、より汎用性の高くて、ユーザーが違和感がない、試しに使ってみるときはデフォルトのまま進めてよいようなデフォルトがよいかと思うので、"Book" は削除して単に "My Title" でどうでしょうか。
それから、"John Doe" は英語圏ではデフォルトの名前として違和感がないかもしれませんが、普通の日本人などには？？？だと思います。これも "My Name" でどうでしょう？